src/tvm/node/node.h

```diff
@@ -41,6 +46,7 @@ class EXPORT AttrVisitor {
   virtual void Visit(const char* key, Type* value) = 0;
   virtual void Visit(const char* key, NodeRef* value) = 0;
   virtual void Visit(const char* key, runtime::NDArray* value) = 0;
+  virtual void Visit(const char* key, runtime::vm::Object* value) = 0;
```

Comment:
Let us just do runtime::Object then

Reply:
I just decided right before you posted this ha. Just pushed a commit to do so.